dandi/download.py

```diff
@@ -825,16 +834,22 @@ def __enter__(self) -> DownloadDirectory:
         ):
             # Pick up where we left off, writing to the end of the file
             lgr.debug(
-                "Download directory exists and has matching checksum; resuming download"
+                "%s - download directory exists and has matching checksum(s) %s; resuming download",
+                self.dirpath,
+                matching_algs,
             )
             self.fp = self.writefile.open("ab")
         else:
             # Delete the file (if it even exists) and start anew
             if not chkpath.exists():
-                lgr.debug("Starting new download in new download directory")
+                lgr.debug(
+                    "%s - starting new download in new download directory", self.dirpath
+                )
             else:
                 lgr.debug(
-                    "Download directory found, but digests do not match; starting new download"
+                    "%s - download directory found, but digests do not match;"
+                    " starting new download",
```

Comment:
Would digests.keys() and self. digests.keys() be useful to know here?

Reply:
I felt like I wondered the same and then dismissed as it would be "too much noise". but now not sure -- it might be... yet to decide